app/assets/src/components/common/CoverageVizBottomSidebar/CoverageVizBottomSidebar.jsx

```diff
@@ -234,15 +272,51 @@ export default class CoverageVizBottomSidebar extends React.Component {
     return {
       referenceAccession,
       referenceLength: currentAccessionData.total_length,
-      alignedContigs: currentAccessionSummary.n_contig,
-      maxAlignedLength: "--",
-      avgCoverageDepth: "--",
-      coverageBreadth: "--",
-      alignedReads: currentAccessionSummary.n_read,
-      avgMismatchedPercent: "--"
+      alignedContigs: currentAccessionSummary.num_contigs,
+      maxAlignedLength: currentAccessionData.max_aligned_length,
+      avgCoverageDepth: currentAccessionData.avg_coverage_depth,
+      coverageBreadth: currentAccessionData.coverage_breadth,
+      alignedReads: currentAccessionSummary.num_reads,
+      avgMismatchedPercent: currentAccessionData.avg_prop_mismatch
     };
   };
 
+  renderContentHeader = () => {
+    const { params } = this.props;
+    const { currentAccessionSummary } = this.state;
+
+    return (
+      <div className={cs.header}>
+        <div className={cs.headerText}>
+          <div className={cs.title}>Coverage for {params.taxonName}</div>
+          <div className={cs.subtitle}>
+            {params.accessionSummaries.length} unique accessions
+          </div>
+        </div>
+        <div className={cs.fill} />
+        <div className={cs.headerControls}>
+          <Dropdown
+            options={this.getAccessionOptions()}
+            value={get("id", currentAccessionSummary)}
+            label="Accession"
+            onChange={this.setCurrentAccession}
+            rounded
+          />
+          <div className={cs.vizLinkContainer}>
+            <a
```

Comment:
I have seen a few times... Should we create a link object to guarantee this is done correctly?